script/run

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Run docker-compose in a container
+#
+# This script will attempt to mirror the host paths by using volumes for the
+# following paths:
+#   * $(pwd)
+#   * $(dirname $COMPOSE_FILE) if it's set
+#   * $HOME if it's set
+#
+# You can add additional volumes (or any docker run options) using
+# the $COMPOSE_OPTIONS environment variable.
+#
+
+
+set -e
+
+VERSION="1.4.0dev"
+# TODO: move this to an official repo
+IMAGE="dnephin/docker-compose:$VERSION"
```

Comment:
I'll open an issue to get this image published to a better repo, and add notes to the release process doc about updating it on release.